### DISCUSSION TITLE
Any plans to move to a shorter package name on npm?

### DISCUSSION THREAD

**Discussion by pjdon** [bikeshaving/crank#81] [General] [answered]
`@bikeshaving/crank` is quite a mouthful. Any plans to move to a shorter package name like other frameworks? Looks like `crankjs` is available on npm.

**Comment by brainkim** [comment, marked as answer]
@pjdon “bikeshaving” is sorta my aspirational consulting/open source organization. I’ve parked crankjs on both github and npm. If it bothers enough people maybe we can move to the crankjs org. But trust me when I say that I got more stuff that might get open sourced and isn’t necessarily related to the crank framework 😉

**Reply by pjdon** [reply]
That's reasonable. I'm interested to see what you have in store for the organization!

If the framework gains a lot of traction and you find the majority of contributions coming from other people, it would be fair to have another think about it.

**Reply by brainkim** [reply]
Of course! Maybe one day crankjs becomes a foundation! Who knows what the future will bring?
